handlers/base.handler.js

```diff
@@ -1,4 +1,4 @@
-const log = require('../initializers/logger');
+const { log } = require('../initializers');
```

Comment:
i changed these imports to make sure that `dotenv` in the `initializers/index` runs, let me know if that's not right